flink-runtime-web/src/main/java/org/apache/flink/runtime/webmonitor/history/HistoryServerArchiveFetcher.java

```diff
@@ -162,7 +166,7 @@ public void run() {
 									String json = archive.getJson();
 
 									File target;
-									if (path.equals("/joboverview")) {
```

Comment:
this change effectively kills backwards compatibility; i.e. a 1.4 HS cannot properly handle 1.5 archives, and a 1.5 HS cannot properly handle 1.4 archives. What is missing is a compatibility check for the old URL.

Reply:
Agree